config/config.ts

```diff
@@ -1,25 +1,32 @@
-const network = '1'; // Ethereum
+const gtonTokenNetwork = '1'; // Ethereum
+const rollupL1NetworkId = '56'; // BSC
 const isDev = process.env.NODE_ENV === 'development';
 const isLive = 'true';
 const isTestnet = false;
 const claimNetwork = '250';
+const bsc = '56';
```

Comment:
Duplicate

Reply:
Also not duplicate - this is used in different places.

We need to make a thorough refactoring to optimize such places